saul-examples/src/main/java/edu/illinois/cs/cogcomp/saulexamples/nlp/SpatialRoleLabeling/Eval/SpRLEvaluation.java

```diff
@@ -0,0 +1,51 @@
+/** This software is released under the University of Illinois/Research and Academic Use License. See
+  * the LICENSE file in the root folder for details. Copyright (c) 2016
+  *
+  * Developed by: The Cognitive Computations Group, University of Illinois at Urbana-Champaign
+  * http://cogcomp.cs.illinois.edu/
+  */
+package edu.illinois.cs.cogcomp.saulexamples.nlp.SpatialRoleLabeling.Eval;
+
+/**
+ * Created by Taher on 2016-09-20.
+ */
+public class SpRLEvaluation {
+    private final String label;
+    private final double precision;
+    private final double recall;
+    private final double f1;
+    private final int labeledCount;
+    private final int predictedCount;
+    public SpRLEvaluation(String label, double precision, double recall, double f1, int labeledCount, int predictedCount) {
+        this.label = label;
+        this.precision = precision;
+        this.recall = recall;
+        this.f1 = f1;
+        this.labeledCount = labeledCount;
+        this.predictedCount = predictedCount;
+    }
+
+    public String getLabel() {
+        return label;
+    }
+
+    public double getPrecision() {
+        return precision;
+    }
+
+    public double getRecall() {
+        return recall;
+    }
+
+    public int getLabeledCount() {
+        return labeledCount;
+    }
+
+    public int getPredictedCount() {
+        return predictedCount;
+    }
+
+    public double getF1() {
+        return f1;
+    }
```

Comment:
This is a container class that holds the evaluation results only. For evaluation I used `TestDiscrete` class from saul here: [evaluate](https://github.com/Rahgooy/saul/pull/2/files/1fea82543ca9fd8e89ab57b0f0e4883b768bc48b#diff-5026aab268091174af3ac8f171fe4cc7R66)